AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/KMSKeystoreOperations.dfy

```diff
@@ -623,14 +625,86 @@ module {:options "/functionSyntax:4" } KMSKeystoreOperations {
       && decryptResponse.Plaintext.Some?
       && 32 == |decryptResponse.Plaintext.value|,
       Types.KeyStoreException(
-        message := "Invalid response from AWS KMS Decrypt: Key is not 32 bytes.")
+        message := ErrorMessages.KMS_DECRYPT_INVALID_KEY_LENGTH_HV1)
     );
 
     output := Success(decryptResponse);
 
   }
 
-  ghost predicate AwsKmsBranchKeyDecryption?(
+  method DecryptKeyForHv2(
```

Comment:
```suggestion
  // TODO-HV-2-M1 : Add a Dafny Test
  method DecryptKeyForHv2(
```

Reply:
I will ignore this suggestion as its already in WIP in [here](https://github.com/aws/aws-cryptographic-material-providers-library/pull/1376/files#diff-71672fd82f87e42c0452e40675f75c6ae02b12630fbcb0629f2d86f8f1f4f1e2R90-R97) and because it takes time for CI run to commit if new commits gets in.